bases/br_ms_sinasc/microdados/table_config.yaml

```diff
@@ -0,0 +1,516 @@
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_uf_mae
+        description: Código da UF de naturalidade da mãe
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_municipio_mae
+        description: Código do município de naturalidade da mãe - IBGE 7 Dígitos
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_pais_residencia
+        description: Código do país de residência
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_municipio_residencia
+        description: ID Município de Residência - IBGE 7 Dígitos
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: data_nascimento_mae
+        description: Data de nascimento da mãe
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: idade_mae
+        description: Idade da mãe
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: escolaridade
+        description: Escolaridade, em anos de estudo concluídos
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: serie_escolar_mae
+        description: Última série escolar concluída da mãe
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: escolaridade_2010
+        description: Escolaridade 2010
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: escolaridade_2010_agr
+        description: Escolaridade 2010 agregada
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: estado_civil
```

Comment:
`estado_civil_mae`

Tem que botar sufixos sempre que a entidade da coluna for diferente da entidade da tabela.